packages/eslint-config/test/__snapshots__/typescript-eslint.spec.js.snap

```diff
@@ -184,6 +196,9 @@ Object {
     "@typescript-eslint/no-misused-promises": Array [
       "error",
     ],
+    "@typescript-eslint/no-mixed-enums": Array [
```

Comment:
New rule: https://typescript-eslint.io/rules/no-mixed-enums/

This will be good to have with the new TS@5 changes to enums.